lms/djangoapps/teams/views.py

```diff
@@ -150,6 +183,39 @@ def get(self, request, course_id):
         }
         return render_to_response("teams/teams.html", context)
 
+    def _serialize_and_paginate(self, pagination_cls, queryset, request, serializer_cls, serializer_ctx):
```

Comment:
It's actually a pretty uncommon use-case for a view to need to serialize and paginate two models.  (This is the only example in the code base I could find).  The usual pattern in DRF 3.1 would be to use a generic view with a serializer class.  DRF would then use or the `pagination_class` attribute of the view (or the default specified in settings) to paginate the results of the view.